### ISSUE TITLE
[1.16.5 server] Server crash when reviving someone

### ISSUE THREAD

**Issue by Watherfoxx** [Cycro54/Revive_Me#20]
- Kill someone
- Revive him (the revive must have a food cost in the config)
- Sometimes the revive bar goes beyond its limits and the server crashes

Server crash-report:
```
---- Minecraft Crash Report ----
// Quite honestly, I wouldn't worry myself about that.

Time: 12/14/24, 4:28 PM
Description: Ticking player

java.lang.NoSuchMethodError: 'void net.minecraft.util.FoodStats.func_75119_b(float)'
	at invoker54.reviveme.common.event.FallenTimerEvent.takeFromReviver(FallenTimerEvent.java:114) ~[reviveme:1.6.16] {re:classloading}
	at invoker54.reviveme.common.event.FallenTimerEvent.TickProgress(FallenTimerEvent.java:90) ~[reviveme:1.6.16] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_173_FallenTimerEvent_TickProgress_PlayerTickEvent.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-4.0.0.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:303) ~[CatServer.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[CatServer.jar:?] {}
	at net.minecraftforge.fml.hooks.BasicEventHooks.onPlayerPreTick(BasicEventHooks.java:85) ~[forge:?] {re:classloading}
	at net.minecraft.entity.player.PlayerEntity.func_70071_h_(PlayerEntity.java:205) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:computing_frames,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.entity.player.ServerPlayerEntity.func_71127_g(ServerPlayerEntity.java:504) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:antiqueatlas-common.mixins.json:MixinPlayerEntityDeath,pl:mixin:A}
	at net.minecraft.server.management.PlayerList.func_72367_e(PlayerList.java:445) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:antiqueatlas-common.mixins.json:MixinPlayerManager,pl:mixin:APP:create.mixins.json:PlayerListMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.server.management.PlayerList.disconnect(PlayerList.java:428) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:antiqueatlas-common.mixins.json:MixinPlayerManager,pl:mixin:APP:create.mixins.json:PlayerListMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.network.play.ServerPlayNetHandler.func_147231_a(ServerPlayNetHandler.java:1521) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.network.NetworkManager.func_179293_l(NetworkManager.java:381) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:160) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:1074) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:336) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:943) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:775) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:265) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at java.lang.Thread.run(Thread.java:840) [?:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at invoker54.reviveme.common.event.FallenTimerEvent.takeFromReviver(FallenTimerEvent.java:114) ~[reviveme:1.6.16] {re:classloading}
	at invoker54.reviveme.common.event.FallenTimerEvent.TickProgress(FallenTimerEvent.java:90) ~[reviveme:1.6.16] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_173_FallenTimerEvent_TickProgress_PlayerTickEvent.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-4.0.0.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:303) ~[CatServer.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[CatServer.jar:?] {}
	at net.minecraftforge.fml.hooks.BasicEventHooks.onPlayerPreTick(BasicEventHooks.java:85) ~[forge:?] {re:classloading}
	at net.minecraft.entity.player.PlayerEntity.func_70071_h_(PlayerEntity.java:205) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:computing_frames,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:A,pl:runtimedistcleaner:A}
-- Player being ticked --
Details:
	Entity Type: minecraft:player (net.minecraft.entity.player.ServerPlayerEntity)
	Entity ID: 1616
	Entity Name: 7Romain7
	Entity's Exact location: -613.95, 77.00, -18.46
	Entity's Block location: World: (-614,77,-19), Chunk: (at 10,4,13 in -39,-2; contains blocks -624,0,-32 to -609,255,-17), Region: (-2,-1; contains chunks -64,-32 to -33,-1, blocks -1024,0,-512 to -513,255,-1)
	Entity's Momentum: 0.00, -0.08, 0.00
	Entity's Passengers: []
	Entity's Vehicle: ~~ERROR~~ NullPointerException: Cannot invoke "net.minecraft.entity.Entity.toString()" because the return value of "net.minecraft.entity.Entity.func_184187_bx()" is null
Stacktrace:
	at net.minecraft.entity.player.ServerPlayerEntity.func_71127_g(ServerPlayerEntity.java:504) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:antiqueatlas-common.mixins.json:MixinPlayerEntityDeath,pl:mixin:A}
	at net.minecraft.server.management.PlayerList.func_72367_e(PlayerList.java:445) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:antiqueatlas-common.mixins.json:MixinPlayerManager,pl:mixin:APP:create.mixins.json:PlayerListMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.server.management.PlayerList.disconnect(PlayerList.java:428) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:antiqueatlas-common.mixins.json:MixinPlayerManager,pl:mixin:APP:create.mixins.json:PlayerListMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.network.play.ServerPlayNetHandler.func_147231_a(ServerPlayNetHandler.java:1521) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:A}
	at net.minecraft.network.NetworkManager.func_179293_l(NetworkManager.java:381) ~[?:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:160) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:1074) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:336) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:943) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:775) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240783_a_(MinecraftServer.java:265) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at java.lang.Thread.run(Thread.java:840) [?:?] {}
```

**Comment by Cycro54** [comment]
Now this has something to do with me using a method that only exists on the client, but I've luckily already fixed this during my extended bug testing. Ty for telling me though, it should be working after I'm done fully testing

**Comment by Cycro54** [comment]
Fixed in revive-me-1.16.5-1.6.24
